app/src/interfaces/_system/system-fields/system-fields.vue

```diff
@@ -28,7 +28,13 @@
 				</button>
 			</template>
 
-			<v-field-list :disabled-fields="value" :collection="collectionName" @select-field="addField" />
```

Comment:
This event rename is a breaking change that wasn't applied to all other uses of `v-field-select`

Reply:
Also changing the event type from `string` to `string[]` makes it a bit of a tricky refactor 😁